### ISSUE TITLE
Photoshop CC 2015 OpenEXR write options

### ISSUE THREAD

**Issue by GiantCowFilms** [davidmilligan/PhotoshopBatchHDR#3]
When set to OpenEXR, every time a merge is complete, it stops until an option for compression has been chosen for saving the file. This is highly annoying, as I had to perform 62 merges, induvidually clicking the box each time. Is there a way for a default to be set in advance to stop this from re occuring? I will make an attempt to investigate myself, however I have no experience with Photoshop scripting. Could this be fixed?

**Comment by Linwood-F** [comment]
@GiantCowFilms  did you find a solution, I am trying this as well, and also never wrote PS scripts, and this is rather defeating the purpose.

**Comment by GiantCowFilms** [comment]
No, Due to other reasons I stopped using Photoshop for this job.... and didn't bother.
